LocalSettings.php

```diff
@@ -3320,23 +3323,25 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'bureaucrat',
 				'banned',
+				'founder',
 			),
 			'moderator' => array(
 				'autopatrolled',
 				'confirmed',
 			),
 			'sysop' => array(
 				'moderator',
-			),
-			'bureaucrat' => array(
-				'moderator',
 				'confirmed',
 				'autopatrolled',
-				'rollback',
+				'rollbacker',
 				'bot',
+				
+			),
+			'bureaucrat' => array(
```

Comment:
I'm still trying to learn how to code directly as Phabricator is so slow. I'll get there eventually.